src/zc/buildout/buildout.py

```diff
@@ -251,15 +253,17 @@ def _print_annotate(data, verbose, chosen_sections, basedir):
 
 
 def _unannotate_section(section):
+    raw = {}
     for key in section:
-        section[key] = section[key].value
-    return section
+        raw[key] = section[key].value
+    return raw
```

Comment:
Why not use a dict comprehension?

```
    return {key: value.value for key, value in section.items()}
```

Reply:
I did not use them often enough. Thanks for pointing this.